src/eth/filter.yaml

```diff
@@ -8,6 +8,9 @@
     name: Filter Identifier
     schema:
       $ref: '#/components/schemas/uint'
+  errors:
+    - code: -32000
+      message: 'invalid from and to block combination: from > to'
```

Comment:
Can you please change to `invalid block range params`

Reply:
I'll also change this one in ethereum/go-ethereum#28386